src/client/src/pages/settings/admin/settingsHeaderContext.tsx

```diff
@@ -0,0 +1,29 @@
+import { createContext, FC, PropsWithChildren, useMemo, useState } from "react";
+
+export interface SettingsHeaderContextProps {
+  headerTitle: string;
+  setHeaderTitle: (editingEnabled: string) => void;
```

Comment:
REQ: Parameter-Name ist hier nicht ganz korrekt, `title` wäre hier passender.

Reply:
War auch mein erster Ansatz, ich finde es allerdings überall wo man den Kontext dann verwendet klarer so. 
Z.B. in einer Detailansicht eine Method "setHeaderTitle" zu brauchen ist für mich klarar als "setTitle".

Reply:
@MiraGeowerkstatt Das ist OK so, ich meine hier den Parameter `editingEnabled` - dieser passt nicht.